app.ts

```diff
@@ -0,0 +1,226 @@
+import { data } from "jquery";
+
+namespace onboardproject {
+	module onboardprojects {
+
+		//Load Variable declarations
+		let Load = 50;
+
+		// Previous Variable declarations
+		let previous: number[];
+
+		//Previous Process Variable declarations
+		let previousprocess: number;
+
+		// Trigger async function
+		async function getRecordCountCall(): Promise<number> {
+			const response = await fetch('http://localhost:2050/recordCount');
+			// let promise = new Promise((res, rej) => {
+			// 	setTimeout(() => res("Now it's done!"), 50)
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		// Trigger async function
+		async function getColumnNamesCall(): Promise<string[]> {
+			const response = await fetch('http://localhost:2050/columns');
+			// let promise = new Promise((res, rej) => {
+			// 	setTimeout(() => res("Now it's done!"), 50)
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		// Trigger async function
+		async function getRecordsCall(fromID: number, toID: number): Promise<string[][]> {
+			const response = await fetch(`http://localhost:2050/records?from=${(fromID)}&to=${(toID)}`);
+			// let promise = new Promise((res, rej) => {
+			// 	setTimeout(() => res("Now it's done!"), 50)
+			// });
+			if (!response.ok) {
+				const message = `An error has occured: ${response.status}`;
+				throw new Error(message);
+			}
+			return response.json();
+		}
+
+		//region Data Loading methods
+		async function LoadRecordsData(fromID: number, toID: number): Promise<number[]> {
+			const recordsvalue = await getRecordsCall(fromID, toID);
+			let DisplayContent = '';
+			for (const record of recordsvalue) {
+				DisplayContent += `<tr id="table-row-${record[0]}">`;
+				for (const column of record) {
+					DisplayContent += `<td align="center">${column}</td>`;
+				}
+				DisplayContent += '</tr>';
+				$("#wrapper-table-content-body").empty();
+				$("#wrapper-table-content-body").append(DisplayContent);
+				throw new Error("Error No data");
+			}
+			return [fromID, toID];
+		}
+
+		// Load Records Function
+		function RecordsFromCursor(cursor: number[]): Promise<number[]> {
```

Comment:
You should rather take in two variables, sorting is not nice, and very expensive when you only care about two of the numbers in the array.